Configuration/TCA/tx_examples_haiku.php

```diff
@@ -157,6 +164,7 @@
                 ],
             ],
         ],
+        /*
```

Comment:
Hm, why not drop instead? Also is this related to backend `UriBuilder` or just a general update?

Reply:
The problems in the haiku TCA throwed errors when trying to call the corresponding backend link functions so I fixed them here

Reply:
OK, but then we can probably drop this altogether?

Reply:
ok, did so just now

Reply:
done